### DISCUSSION TITLE
ERROR POSTCSS: failed to transform "/scss/main.css" (text/css): Error: Loading PostCSS Plugin failed: Cannot find module 'autoprefixer'

### DISCUSSION THREAD

**Discussion by jpanico** [google/docsy#2083] [Q&A] [answered]
Hi, 

I'm new to Docsy (and Hugo). Trying to build the example site for the very first time. I'm getting this error:

```
     Start building sites … 
     hugo v0.134.2-1c74abd26070b0c12849550c974a9f3f1e7afb06+extended linux/amd64 BuildDate=2024-09-10T10:46:33Z VendorInfo=gohugoio
     
     ERROR POSTCSS: failed to transform "/scss/main.css" (text/css): Error: Loading PostCSS Plugin failed: Cannot find module 'autoprefixer'
     Require stack:
     - /workspace/postcss.config.js
     
     (@/workspace/postcss.config.js)
         at load (/usr/local/lib/node_modules/postcss-cli/node_modules/postcss-load-config/src/plugins.js:28:11)
         at async Promise.all (index 0)
         at async plugins (/usr/local/lib/node_modules/postcss-cli/node_modules/postcss-load-config/src/plugins.js:57:12)
         at async processResult (/usr/local/lib/node_modules/postcss-cli/node_modules/postcss-load-config/src/index.js:39:14)
         at async Promise.all (index 0)
     Total in 6667 ms
```

The strange thing is that autoprefixer package is installed in the same location as the postcss-cli package, which is presumably trying to call it:

```
     + npm -g list
     /usr/local/lib
     ├── autoprefixer@10.4.20
     ├── postcss-cli@11.0.0
     └── postcss@8.4.47
```

```
     + npx autoprefixer --version
     autoprefixer 10.4.20
```

Any clues much appreciated.

thanks,

Joe

**Comment by deining** [comment, marked as answer]
This issue was already raised in #1586.
From your logs I can see you installed autoprefixer and postcss globally, but docsy expects them locally, this may be the cause for your troubles.

Possible solution: can you please check if there is a `package.json in the root directory of your site? If not, can you please run:

```
npm init
```

Then answer the questions in the command line questionnaire.
Afterwards, issue:

```
npm install -D autoprefixer
npm install -D postcss-cli
```

Now you should be able to successfully run:

```
hugo server
```

Can you please try that and report back whether this cures your problem?

**Reply by jpanico** [reply]
hi Deining,

Installing autoprefixer/postcss-cli locally worked! 

Thanks for your help.
